react/src/pages/InfoPage.js

```diff
@@ -188,10 +193,34 @@ const InfoPage = () => {
         </InfoBox>
         <InfoBox info={info} title="Resources">
             <InfoRow name={"Foursight Server"} value={info.get("server.foursight")} monospace={true} copy={true} size="2" />
-            <InfoRow name={"Portal Server"} value={info.get("server.portal")} monospace={true} copy={true} size="2" portalCertificate={true} />
-            <InfoRow name={"ElasticSearch Server"} value={info.get("server.es")} monospace={true} copy={true} size="2" />
-            <InfoRow name={"RDS Server"} value={info.get("server.rds")} monospace={true} copy={true} size="2" />
-            <InfoRow name={"SQS Server"} value={info.get("server.sqs")} monospace={true} copy={true} size="2" />
+            <InfoRow name={"Foursight"} value={header.resources?.foursight} monospace={true} copy={true} size="2" />
```

Comment:
All this is just a reordering of sections on the INFO page.

Reply:
Thanks for that tip.